rules-emerging-threats/2025/Exploits/CVE-2025-54309/proc_creation_win_exploit_cve_2025_54309.yml

```diff
@@ -0,0 +1,53 @@
+title: Potential Exploitation of CrushFTP RCE Vulnerability (CVE-2025-54309)
+id: 0fdc7c7f-c690-4217-9ae3-31f5156eed72
+status: experimental
+description: Detects suspicious child processes created by CrushFTP. It could be indication of exploitation of RCE vulnerability such as CVE-2025-54309.
```

Comment:
```suggestion
description: Detects suspicious child processes created by CrushFTP. It could be an indication of exploitation of a RCE vulnerability such as CVE-2025-54309.
```